includes/class-indieauth-debug.php

```diff
@@ -31,7 +31,7 @@ public function indieauth_allow_localhost( $r, $url ) {
 	public function log_rest_api_errors( $result, $server, $request ) {
 		$request_route = $request->get_route();
 		$result_route  = $result->get_matched_route();
-		$routes        = array( 'indieauth/1.0/auth', '/indieauth/1.0/token' );
+		$routes        = array( 'indieauth/1.0/auth', '/indieauth/1.0/token', 'indieauth/1.0/ticket' );
```

Comment:
Won't be a showstopper, but this feels like it may be getting primed for a dedicated method for retrieving routes. Could also potentially get a filter for 3rd party integration.

Reply:
Good point. This was written a long time ago. I should have it retrieve all indieauth namespace routes.

Reply:
I'll likely save the idea for a future PR though.